packages/core/src/accordion/accordion.element.ts

```diff
@@ -37,6 +37,7 @@ import { styles } from './accordion.element.css.js';
  * @cssprop --border-color
  * @cssprop --border-width
  * @cssprop --border-radius
+ * @cssprop --icon-color
```

Comment:
is this cssprop in use?

Reply:
It's what we use to customize the caret icon color in the accordion. Where it gets applied specifically is in the header.  Let me move to annotation to header. 🙂

Reply:
I realized I still had a bug in this while updating the PR. Latest code changes reflect the following default / customizability for icon colors in accordion header:
- the caret icon color defaults to the color of the header
- app dev can override the icon color at the header scope using the `--icon-color` custom property